ledger-core-ethereum/src/ethereum/explorers/EthereumLikeTransactionParser.cpp

```diff
@@ -30,6 +30,7 @@
 
 #include <core/math/Base58.hpp>
 #include <core/utils/DateUtils.hpp>
+#include <core/database/Migrations.hpp>
```

Comment:
What do you need this for?

Reply:
https://github.com/LedgerHQ/lib-ledger-core/pull/520/files/eea3e94c9ca0ac2eba1a459976e3969c247fadd5#diff-81857d29400a5347ca448911bce5bbb4R43 MAX_LENGTH_VAR_CHAR it seems

Reply:
Exactly !